src/expandableCalendar/WeekCalendar/index.tsx

```diff
@@ -23,6 +23,7 @@ const APPLY_ANDROID_FIX = constants.isAndroid && constants.isRTL;
 export interface WeekCalendarProps extends CalendarListProps {
   /** whether to have shadow/elevation for the calendar */
   allowShadow?: boolean;
+  disableDaySelection?: boolean;
```

Comment:
Is `disableDaySelection` prop only relevant to the Week and WeekCalendar components? Can't it be relevant to the CalendarProps?

Reply:
our usage of display day selection is only relevant for weekly display with custom number of days per week, in the display the knob is hidden and you cant toggle between week view and month view

Reply:
Then it shouldn't be a public prop. See my comment below